firmware/controllers/system/timer/single_timer_executor.cpp

```diff
@@ -127,6 +127,13 @@ void SingleTimerExecutor::executeAllPendingActions() {
 	do {
 		efitick_t nowNt = getTimeNowNt();
 		didExecute = queue.executeOne(nowNt);
+
+		// If locked (ie, interrupts disabled), briefly enable/disable them
+		// to flush any pending interrupts
+		// This is safe to do here because it's the same as if the just executed
```

Comment:
does this great change belong in "compensate injector flow based on fuel pressure" PR? let's please separate?

Reply:
oops, that wasn't supposed to sneak in